src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -46,12 +46,12 @@ This section provides a tutorial on how to internationalize
 a Flutter application, along with any additional setup that a
 target platform might require.
 
-### Setting up an internation&shy;alized app: the flutter<wbr>_localizations package {#setting-up}
+### Setting up an internation&shy;alized app: the Flutter<wbr>_localizations package {#setting-up}
```

Comment:
`flutter_localizations` here is correct, since that's the package name: https://github.com/flutter/flutter/tree/master/packages/flutter_localizations

Reply:
Ah, then we code font it.